### PR TITLE
Fix/description and traces

### PR DESCRIPTION
**Pull request by SerGeRybakov**
- description of scaled dimensions fixed
- client instantiation errors stack traces fixed
- added manual CI workflow for docs building and publishing
- docs for deker-shell and deker-server-adapters updated

### CODE REVIEW

File: deker/ABC/base_factory.py [hunk 1]
```diff
@@ -52,8 +52,11 @@ def __del__(self) -> None:
     @abstractmethod
     def close(self) -> None:
         """Close own executor."""
-        if self.executor and self._own_executor:
-            self.executor.shutdown(wait=True, cancel_futures=False)
+        try:
+            if self.executor and self._own_executor:
+                self.executor.shutdown(wait=True, cancel_futures=False)
+        except Exception:
```
Comment (matveyvarg) [inline deker/ABC/base_factory.py:58]: There should be specific Exception. We still want to fail on unexpected things.

File: deker/client.py [hunk 2]
```diff
@@ -330,15 +330,20 @@ def calculate_storage_size(self, collection_name: str = "") -> StorageSize:
...
+            if self.__adapter:
+                self.__adapter.close()
+                self.__adapter = None
+            if self.__factory:
+                self.__factory.close()
+                self.__factory = None
+            if self.__ctx:
+                self.__ctx.is_closed = True
+            self.__is_closed = True
+        except Exception as e:
```
Comment (matveyvarg) [inline deker/client.py:343]: Same as in previous comment.

Comment (SerGeRybakov) [reply deker/client.py]: [52b8348](https://github.com/openweathermap/deker/pull/15/commits/52b8348e12680f1d16548cbd43d8635fe2f109fb)

File: deker/ABC/base_factory.py (hunk 1 above)
Comment (SerGeRybakov) [reply deker/ABC/base_factory.py]: [52b8348](https://github.com/openweathermap/deker/pull/15/commits/52b8348e12680f1d16548cbd43d8635fe2f109fb)

**Review by matveyvarg — DISMISSED** [review]

**Review by matveyvarg — APPROVED** [review]

**Pull request merged**